### PR TITLE
Sunset mode

### PR DESCRIPTION
**Pull request by faboweb**
image

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5fbc4cad5783b68de7124103--lunieio.netlify.app

**Review by Bitcoinera — DISMISSED** [review]

File: app/src/components/account/ManageAccountsModal.vue [hunk 1]
```diff
@@ -1,8 +1,21 @@
 <template>
-  <SessionFrame icon="build" hide-back>
-    <div class="select-account">
-      <AccountList :accounts="accounts" is-select-account />
-    </div>
+  <SessionFrame icon="exit_to_app" hide-back>
+    <p>
+      The Lunie Web Wallet has sunset. Thank you for your support!
```
Comment (jbibla) [inline app/src/components/account/ManageAccountsModal.vue:4]:
```suggestion
      🌅 The Lunie web wallet has sunset. Thank you for your support!
```

File: app/src/components/account/ManageAccountsModal.vue [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+    </p>
+    <br />
+    <a href="http://help.lunie.io/en/collections/2624438-lunie-address-migration-guides">
+      Migration Guides
+    </a>
+    <br/>
+    <template v-if="accounts.length > 0">
+      <p>
+        Extract your seeds by clicking on the 3 dots, right side of your key.
```
Comment (jbibla) [inline app/src/components/account/ManageAccountsModal.vue:13]:
```suggestion
        Extract your seeds by clicking on the 3 dots below.
```

File: app/src/components/account/ManageAccountsModal.vue [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
-    <div class="select-account">
-      <AccountList :accounts="accounts" is-select-account />
-    </div>
+  <SessionFrame icon="exit_to_app" hide-back>
+    <p>
+      The Lunie Web Wallet has sunset. Thank you for your support!
+    </p>
+    <br />
+    <a href="http://help.lunie.io/en/collections/2624438-lunie-address-migration-guides">
+      Migration Guides
```
Comment (jbibla) [inline app/src/components/account/ManageAccountsModal.vue:8]:
```suggestion
      Here are some helpful migration guides.
```

File: app/src/components/account/ManageAccountsModal.vue [hunk 4]
```diff
@@ -1,8 +1,21 @@
 <template>
-  <SessionFrame icon="build" hide-back>
-    <div class="select-account">
-      <AccountList :accounts="accounts" is-select-account />
-    </div>
+  <SessionFrame icon="exit_to_app" hide-back>
+    <p>
+      The Lunie Web Wallet has sunset. Thank you for your support!
+    </p>
+    <br />
```
Comment (jbibla) [inline app/src/components/account/ManageAccountsModal.vue:6]:
```suggestion

```

**Comment by jbibla** [comment]
this is still only going to be visible on mobile, not web, right?

**Comment by faboweb** [comment]
> this is still only going to be visible on mobile, not web, right?

this is going to be on web and mobile

**Comment by jbibla** [comment]
you can't reveal your seed on web so the message and 3 dots should be hidden. there is also no need to show the account list on web... but we talked about a redirect. so not sure what is best...

**Comment by faboweb** [comment]
I get it. Let's just do the redirect on web

**Comment by jbibla** [comment]
> I get it. Let's just do the redirect on web

sure, sounds good. 

---

bunch of checks are failing

**Comment by faboweb** [comment]
it doesn't matter that the checks are failing imo

**Pull request closed without merging**